### PR TITLE
Add tip for zooming in the bezier curve editor

### PR DESCRIPTION
**Pull request by kidManuel**
After spending quite a bit of time trying to figure out how to scale the Y axis of the Bezier curve navigator (and getting my answer from Discord) I decided to do my first contribution to the docs by adding a tip to the animation tutorial.
ESL, so very open to grammatical corrections.

### CODE REVIEW

File: tutorials/animation/animation_track_types.rst [hunk 1]
```diff
@@ -72,7 +72,14 @@ editor, click the curve icon to the right of the animation track.
 .. image:: img/bezier_curve_icon.webp
 
 In the editor, keys are represented by filled diamonds and the outlined
-diamonds connected to them by a line control curve's shape.
+diamonds connected to them by a line control curve's shape.
```
Comment (AThousandShips) [inline tutorials/animation/animation_track_types.rst:75]:
```suggestion
diamonds connected to them by a line control curve's shape.
```

**Review by mhilbrunner — APPROVED** [review]

File: tutorials/animation/animation_track_types.rst [hunk 2]
```diff
@@ -74,6 +74,13 @@ editor, click the curve icon to the right of the animation track.
 In the editor, keys are represented by filled diamonds and the outlined
 diamonds connected to them by a line control curve's shape.
 
+.. tip::
+
+    For better precision while manually working with curves, you might want to alter
+    the zoom levels of the editor. You can do this with your mouse wheel, or using
+    the track on the bottom right of the editor to zoom in on the time axis, 
+    or with :kbd:`Ctrl + Alt + Mouse wheel` to zoom in the values range of the animation.
```
Comment (skyace65) [inline tutorials/animation/animation_track_types.rst:82]:
```suggestion
    the zoom levels of the editor. The slider on the bottom right of the editor can be used to
    zoom in and out on the time axis, you can also do that with :kbd:`Ctrl + Shift + Mouse wheel`.
    Using :kbd:`Ctrl + Alt + Mouse wheel` will zoom in and out on the Y axis
```

Phrasing needs to be adjusted to be clearer. Also I've never seen a slider called a track before.

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: tutorials/animation/animation_track_types.rst (hunk 2 above)
Comment (kidManuel) [reply tutorials/animation/animation_track_types.rst]: Good review. Thanks. Sorry about the slider thing.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution! And thanks for the reviews peeps :)

**Pull request merged**